README.md

```diff
@@ -1,20 +1,43 @@
 # \*IPs - Star Improvement Proposals
 *by Zachary Belford, Shane Jonas, Donald McIntyre, Yaz Khoury, Anthony Lusardi, Tomaz Kariz*
 
-Star Improvement Proposals are an open standard for contributors to propose changes, additions, or deprecations of core protocol specifications.
+Star Improvement Proposals is an extension of the BIP/EIP/ECIP process, with some changes to allow for decentralization of the process. Its fundamental change involves determinism of proposal numbers and content-based versioning. 
```

Comment:
If only the preamble is getting hashed, that doesn't seem exactly seem to me like "content-based versioning" so much as "title based versioning".

Reply:
The thought here is that the preamble should contain sufficient information to identify itself from another proposal, otherwise, how confusing would that be??

Reply:
Would it be better if named as suggested? or something to the effect of 'preamble content versioning' ?

Reply:
@whilei @BelfordZ in the spirit of unblocking Donald so he has an updated proposal for his Medium Post, I'll be resolving this conversation and suggesting we make any new PR for the ideas you had.